SpatialGDK/Source/SpatialGDK/Public/EngineClasses/SpatialVirtualWorkerTranslator.h

```diff
@@ -25,7 +25,9 @@ class SPATIALGDK_API SpatialVirtualWorkerTranslator
 	// Currently that is only the number of virtual workers desired.
 	bool IsReady() const { return bIsReady; }
 
-	void SetDesiredVirtualWorkerCount(uint32 NumberOfVirtualWorkers);
+	void AddVirtualWorkerIds(const TSet<VirtualWorkerId>& InVirtualWorkerIds);
+	VirtualWorkerId GetLocalVirtualWorkerId() const { return LocalVirtualWorkerId; }
+	PhysicalWorkerName GetLocalPhysicalWorkerName() const { return WorkerId; }
```

Comment:
I wonder what the benefit is for defining `PhysicalWorkerName` type and not using `FString` directly?

Reply:
I think mostly clarity. It bugs me that we call things "Id" and then they're actually strings. So I wanted to be clear that it was a name and a string.

Reply:
Ok, resolving.